assets/user-manifests-bootstrapper/user-manifests-bootstrapper-pod.yaml

```diff
@@ -92,6 +92,21 @@ spec:
           # Replace the global certs configmap here because it's too large to oc apply
           oc create configmap -n openshift-controller-manager openshift-global-ca --from-file ca-bundle.crt=/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem --dry-run=client -o yaml > /tmp/openshift-global-ca
           oc replace -n openshift-controller-manager -f /tmp/openshift-global-ca --force
+          # Create the ClusterVersion resource if it doesn't already exist
+          oc get ClusterVersion version
+          CLUSTER_VERSION_NOT_FOUND=$?
+          if [[ $CLUSTER_VERSION_NOT_FOUND -eq 1 ]]; then
+             apt install uuid-runtime
```

Comment:
We cannot do such an install at runtime  This would violate our security checks.

Reply:
@Evan-Reilly I was hoping you would generate the clusterversion resource as one of the bootstrap resources, but only conditionally apply it here if not found